015_flatpandoc_L84/015_flatpandoc_L84_unsafe.py

```
@@ -0,0 +1,117 @@
+
+try:
+  __version__ = pkg_resources.require("Flask-FlatPages-Pandoc")[0]
+except pkg_resources.DistributionNotFound:
+  __version__ = "0.0-dev"
+
+class FlatPagesPandoc(object):
+  """
+  Class that, when applied to a :class:`flask.Flask` instance,
+  sets up an HTML renderer using pandoc.
+  """
+  def __init__(self, source_format, app=None, pandoc_args=[],
```

Ruff (0.14.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)